Tp2

cainellicamila opened the pull request.

cainellicamila committed:
Add files via upload

cainellicamila committed:
Add files via upload

cainellicamila committed:
Add files via upload

Franr commented on Tp2-1.py:
```diff
@@ -0,0 +1,20 @@
+#Escribir una clase llamada rectángulo que contenga una base y una altura, y que contenga un
+#método que devuelva el área del rectángulo.
+
+class rectanulo ():
```
Rectangulo

Franr commented on Tp2-1.py:
```diff
@@ -0,0 +1,20 @@
+
+class rectanulo ():
+
+    base = 0
```
no usar variables de clases

Franr commented on Tp2-1.py:
```diff
@@ -0,0 +1,20 @@
+        return self.area
+
+rec = rectanulo ()
+rec.base = 4
```
pasar parametros al crear la instancia

Franr commented on Tp2-2.py:
```diff
@@ -0,0 +1,21 @@
+
+circu = circulo()
+circu.radio = 5
+print ('El área del circulo es:',circu.calcarea())
```
faltan asserts

Franr commented on Tp2-2.py:
```diff
@@ -0,0 +1,21 @@
+from math import pi
+
+class circulo:
+    radio = 0
```
no usar variables de clases

Franr commented on Tp2-2.py:
```diff
@@ -0,0 +1,21 @@
+        return self.peri
+
+circu = circulo()
+circu.radio = 5
```
pasar parametros al crear la instancia

Franr commented on Tp2-2.py:
```diff
@@ -0,0 +1,21 @@
+'''
+from math import pi
+
+class circulo:
```
por convención el nombre de las clases empieza con mayúscula

Franr reviewed (changes requested):
Corregir los mismos errores en todos los ejercicios:
- Agregar asserts para todos los metodos/casos
- No usar variables de clases
- Los nombres de las clases van en CamelCase
- Los parámetros de una instancia se pasan a través del constructor, no despues de crearla

cainellicamila committed:
Add files via upload

Franr commented on Tp2_4.py:
```diff
@@ -0,0 +1,42 @@
+
+per=Estudiante('Camila', 20, 'F', 48,162, 40119128, 'ISI', 2015, 41, 24)
+
+per.avance()
```
faltarian asserts aca

Franr commented on Tp2_4.py:
```diff
@@ -0,0 +1,42 @@
+# - avance(): indica que porcentaje de la carrera tiene aprobada.
+# - edad_ingreso(): indica que edad tenia al ingresar a la carrera (basándose en el año actual).
+
+class Persona:
```
importarla desde el ejercicio 3

Franr commented on Tp2-5.py:
```diff
@@ -0,0 +1,32 @@
+    dic = {}
+    for e in estu:
+        dic[e] = estu.count(e)
+    assert dic['ISI'] == 3
```
los asserts van afuera de la funcion

Franr commented on Tp2-5.py:
```diff
@@ -0,0 +1,32 @@
+    assert dic['IQ'] ==2
+    assert dic['ISI'] != 5
+    assert dic['IQ'] != 3
+    print(dic)
```
debe devolver, no imprimir

cainellicamila committed:
Add files via upload

(El Tp2_3 esta exactamente igual pero como para importar la clase en el ej 4 no podia llamarse Tp2-3 le cambie el nombre)

Correcciones ej 4 y 5
